internal/cmd/gocdk/demo_test.go

```diff
@@ -57,6 +58,9 @@ func TestAddDemo(t *testing.T) {
 
 	// Build the binary.
 	exePath := filepath.Join(dir, "add-demo-test")
+	if runtime.GOOS == "windows" {
+		exePath += ".EXE"
```

Comment:
Is it really uppercase `.EXE` these days?

Reply:
Doesn't matter, Windows is case-insensitive.